mantidproject/mantidimaging
# Reset preview view on image preview

Fixes #546 

To test:
- Load data
- Go to operations
- Play around with the crop-cordinates operation and the previews and ensure that it looks correct as you move the ROI on the stack visualiser
